db/migrate/20180118192721_create_glitch_notes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGlitchNotes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :glitch_notes do |t|
+      t.bigint :target_id, null: false
```

Comment:
This probably doesn't need to be `BIGINT`: notes pertain to _mutes_ and _blocks_, and those are still using `INTEGER`-size IDs.

Reply:
Never mind, I lied:

```
mastodon_development=# \d blocks
                                         Table "public.blocks"
      Column       |            Type             |                      Modifiers                      
-------------------+-----------------------------+-----------------------------------------------------
 id                | bigint                      | not null default nextval('blocks_id_seq'::regclass)
 created_at        | timestamp without time zone | not null
 updated_at        | timestamp without time zone | not null
 account_id        | bigint                      | not null
 target_account_id | bigint                      | not null
```